README.md

```diff
@@ -52,12 +55,17 @@ A Python library of exchange calendars meant to be used with [Zipline](https://g
 | Buenos Aires Stock Exchange     | XBUE     | Argentina      | 1.11          | https://www.bcba.sba.com.ar/                                 |
 | Pakistan Stock Exchange         | XKAR     | Pakistan       | 1.11          | https://www.psx.com.pk/                                      |
 
-Calendars marked with an asterisk (*) have not yet been released.
```

Comment:
We don't have any calendars that we're planning to add at this point, so you're right that we should remove the footnote. Thanks!